src/zyre.c

```diff
@@ -277,6 +277,23 @@ zyre_set_endpoint (zyre_t *self, const char *format, ...)
     return zsock_wait (self->actor) == 0? 0: -1;
 }
 
+#ifdef ZYRE_BUILD_DRAFT_API
```

Comment:
@sappo this is causing the linking error - the draft/nondraft is in the headers, and affects the visibility attribute used with the declarations, but in the body we always define the functions